app/helpers/task_list_helper.rb

```diff
@@ -58,7 +58,7 @@ def row_helper(trainee, task)
     when :school_details
       {
         task_name: school_details_title(trainee.training_route),
-        path: edit_trainee_lead_schools_path(trainee),
+        path: edit_trainee_lead_partners_path(trainee),
         confirm_path: trainee_schools_confirm_path(trainee),
         classes: "school-details",
         status: ProgressService.call(
```

Comment:
I think those would have to be refactored to accommodate Lead Partners as they can be either Schools or Providers.